SPECS-SIGNED/grub2-efi-binary-signed-aarch64/grub2-efi-binary-signed-aarch64.spec

```diff
@@ -1,8 +1,8 @@
 %global debug_package %{nil}
 Summary:        Signed GRand Unified Bootloader for aarch64 systems
 Name:           grub2-efi-binary-signed-aarch64
-Version:        2.02
-Release:        26%{?dist}
+Version:        2.06
```

Comment:
Shouldn't this be `2.06-rc1`?

Reply:
Don't believe I can place `-` in the Version number field. I debated to put `2.06~rc1` but wasn't sure what would happen when 2.06 was officially released and our tools did version comparison checks.

Reply:
I'll see if I can construct a small test for this. I would prefer to use `2.06~rc1` here if I can.

Reply:
Right, the tilde makes a lot more sense, sorry.

Reply:
Okay testing shows `2.06` > `2.06~rc1` > `2.02`, which is the intended behavior. Will update this PR with `2.06~rc1`

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_versioning_prereleases_with_tilde

Reply:
Thanks!